### PR TITLE
Add calendar-specific config for maximumEntries and maximumNumberOfDays

### PR DESCRIPTION
**Pull request by Nosrac**
Now you can force specific calendars to use separate settings for maximumNumberOfDays and maximumEntries.

Usable with a config file like:
```
config: {
    calendars : [ {
        url: "webcal://...",
        config: {
            maximumNumberOfDays: 3,
            maximumEntries: 1
        }
    } ]
}

### CODE REVIEW

**Comment by MichMich** [comment]
Thanks for your contribution. To be honest, I think adding a config in a config will create a lot of confusion. I prefer the following setup:
````javascript
config: {
    calendars : [ {
        url: "webcal://...",
        maximumNumberOfDays: 3,
        maximumEntries: 1
    } ]
}
````

**Comment by Nosrac** [comment]
@MichMich Yeah, that's probably better.  I've made the changes and updated the calendar README.

**Comment by MichMich** [comment]
Awesome. Could you also add an updated CHANGELOG.md?

**Comment by Nosrac** [comment]
@MichMich Sure, I wasn't sure what to do with the version number so I bumped it to 2.2.0  Not sure if that's what you wanted or not.

At some point I needed to do a merge which confused me.  Did I accidentally merge with the wrong branch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+<<<<<<< HEAD
```
Comment (roramirez) [inline CHANGELOG.md]: Conflict merge here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+## [2.2.0] - 2017-01-22
+
+### Added
+- Support calendar-specific controls for `maximumNumberOfDays` and `maximumEntries`
+=======
```
Comment (roramirez) [inline CHANGELOG.md]: The same

**Comment by MichMich** [comment]
@Nosrac You probably based this change form an old master branch. This commit would change 42 files, reverting them back to old versions. Could you please rebase your changes and resubmit a PR to the develop branch? Thanks!

**Pull request closed without merging**